src/main/java/hellos/Hellos.java

```diff
@@ -33,4 +34,8 @@ private String kkSaysHello() {
     return "KK says 'Hello!'\n";
   }
 
+  private String susanSaysHi() {
+    return "Susan says 'Welcome!'\n";
+  }
+
```

Comment:
Given that this matches the format of the rest and that the tests past, I think that you're ready to go!

Reply:
As before, the method returns a correctly formatted string, utilizing a name, "says", a greeting, and a tag for a new line.